data/campaigns/Under_the_Burning_Suns/units/quenoth/Fighter.cfg

```diff
@@ -11,20 +11,18 @@
     {TRAIT_RESILIENT}
     image=units/quenoth/fighter.png
     profile="portraits/quenoth/fighter.webp"
-    hitpoints=30
+    hitpoints=33
     movement_type=quenoth_foot
     movement=5
     experience=40
     level=1
     alignment=lawful
     advances_to=Quenoth Warrior,Quenoth Flanker
-    cost=15
+    cost=12
     usage=fighter
     description= _ "Long ago, during more prosperous times, elven warriors favored the use of swords as more elegant, versatile weapons compared to other melee armaments. However, in recent times, the dearth of supplies for smithing has reduced the availability of blade-crafting, necessitating the fashioning of cheaper, more easily repaired weaponry. Quenoth Fighters are remarkably skilled in coordinated groups, where they can counteract the lack of more sophisticated weaponry by forming robust spear ranks capable of repelling nearly any frontal assault."
```

Comment:
This text describes the Formation ability, and needs to be updated too.

With a redesign that changes the signature ability, I think it would be better to make this a new unit line; as working title maybe "Quenoth Duelist". While the Quenoth Fighter wouldn't be in the campaign any more, it could resurface in Ageless or other campaigns.

Reply:
Even if the ability isn't around, it would be helpful to rename the units. That would make it easier to refer to in walkthroughs that will still need to cater to 1.16 players even after 1.18 is out.

The ability itself is likely to live on in the unit tests - while it isn't found there yet, there's parts of the C++ code that I want to be tested by something in mainline. If I can no longer find coverage for that C++ in mainline then it'll be added to the tests.

Reply:
Honestly I like the current simple name and I don't think that the change is necessary. Also this with this reasoning I would have to rename most of the units because well, most of them changed a lot. 

Well it being in unit tests is outside of reach of your usual gamer. :P

Reply:
I also prefer leaving the name fighter, but if it's really that big of a problem for code then I guess we can change it.